### PR TITLE
Add mock api for county/zipcode data

### PR DESCRIPTION
**Pull request by daningenthron**
Adds a json-server mock API containing the county and zipcode json files, to allow for an easier transition once the UI is synced with the back end. API calls are made from ComponentDidMount in App.js.

**Modules added**:
- Axios: For the API call.
- Concurrently: Sets the server up on port 3001; kills it when the app is shut down.

**To Do / Possible Issues**
I initially planned on setting up a separate MockApi class but ultimately found it easier and leaner to keep the logic in App.js. This could be cleaned up; this also may be addressed once Redux is brought into the app.

### CODE REVIEW

**Comment by egabrielse** [comment]
I tried it out locally and it works great! One thing I would suggest is adding concurrently and json-server to the package.json's devDependencies, so that they can be installed easily with "npm install."
`"devDependencies": {
    "concurrently": "^5.3.0",
    "json-server": "^0.16.3"
  },`

**Review by mcdevittbass — APPROVED** [review]
Thanks, Dan!

File: package.json [hunk 1]
```diff
@@ -42,5 +44,7 @@
   "devDependencies": {
     "concurrently": "^5.3.0",
     "json-server": "^0.16.3"
```
Comment (egabrielse) [inline package.json:46]: I think this line needs a comma after it. And the line below should have a colon separating the name and version number

Comment (daningenthron) [reply package.json]: Fixed!

**Review by egabrielse — APPROVED** [review]
Looks good to go! Good job!

**Pull request merged**